lib/src/DateRangePicker/DateRangePickerDay.tsx

```diff
@@ -96,19 +96,20 @@ const useStyles = makeStyles(
   { name: 'MuiPickersDateRangeDay' }
 );
 
-export const PureDateRangeDay = ({
-  className,
-  day,
-  inCurrentMonth,
-  isEndOfHighlighting,
-  isEndOfPreviewing,
-  isHighlighting,
-  isPreviewing,
-  isStartOfHighlighting,
-  isStartOfPreviewing,
-  selected,
-  ...other
-}: DateRangeDayProps) => {
+export const PureDateRangeDay = (props: DateRangeDayProps) => {
+  const {
+    className,
+    day,
+    inCurrentMonth,
+    isEndOfHighlighting,
+    isEndOfPreviewing,
+    isHighlighting,
+    isPreviewing,
+    isStartOfHighlighting,
+    isStartOfPreviewing,
+    selected,
+    ...other
+  } = props;
```

Comment:
What the original motiviation for this kind of change? Is this doing automatically?

Reply:
The first-order motivation is to match the convention of the main repository. The second-order motivation was to allow quick scanning of the signature of the function. If you see "props" => it means it's a React component.